src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelAPITests.cs

```diff
@@ -4450,6 +4450,54 @@ public partial class C
             Assert.Equal("DEBUG", model.GetConstantValue(root.DescendantNodes().OfType<InvocationExpressionSyntax>().Single()));
         }
 
+        [Theory, CombinatorialData, WorkItem(54437, "https://github.com/dotnet/roslyn/issues/54437")]
+        public void TestVarTuple(NullableContextOptions nullableContextOption)
+        {
+            var source = """
+                var (a1, b1) = ("", 0);
+                """;
+            var options = WithNullable(TestOptions.DebugExe, nullableContextOption);
+            var comp = CreateCompilation(source, options: options, parseOptions: TestOptions.Regular9);
+            comp.VerifyDiagnostics();
+
+            var tree = comp.SyntaxTrees.Single();
+            var model = comp.GetSemanticModel(tree);
+            var root = tree.GetCompilationUnitRoot();
+
+            var varNode = root.DescendantNodes().OfType<DeclarationExpressionSyntax>().Single().Type;
+
+            var varTypeInfo = model.GetTypeInfo(varNode);
+            assertTypeInfo(varTypeInfo);
+
+            var varSymbolInfo = model.GetSymbolInfo(varNode);
+            assertSymbolInfo(varSymbolInfo);
+
+            static void assertTypeInfo(TypeInfo typeInfo)
+            {
+                Assert.Equal(CodeAnalysis.NullableAnnotation.None, typeInfo.Nullability.Annotation);
+                Assert.Equal(CodeAnalysis.NullableAnnotation.None, typeInfo.ConvertedNullability.Annotation);
```

Comment:
This is very unexpected to me when nullable is enabled.

It seems that nullable walker doesn't update `TopLevelNullability` for the `BoundConvertedTupleLiteral`, probably there is a missing override and/or call to `SetAnalyzedNullability` in nullable walker.

Reply:
Tuple literals are a known issue with nullability.

Reply:
@333fred Is there an existing issue for that?

Reply:
https://github.com/dotnet/roslyn/issues/35010